packages/components/src/test/tree/controlled/UsePresentationTreeState.test.ts

```diff
@@ -46,6 +46,7 @@ describe("usePresentationTreeState", () => {
     imodel,
     ruleset: rulesetId,
     pagingSize: 5,
+    onNodeLoaded: undefined,
```

Comment:
We is this necessary?

Reply:
I am not sure why but it was necessary, however with the latest changes it was no longer needed.